my.css

```diff
@@ -107,19 +108,20 @@ body {
   display: flex;
   justify-content: center;
   align-items: center;
-  height: 70%;
-  padding-left: 100px;
+  height: 80%;
+  padding-left: 110px;
 }
 
 .collection__img {
-  width: 500px;
-  height: 500px;
+  width: 530px;
+  height: 550px;
 }
 
 .newcollection__footer {
   display: flex;
   justify-content: space-between;
   align-items: center;
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)